builtin/rebase.c

```diff
@@ -852,6 +857,24 @@ int cmd_rebase(int argc, const char **argv, const char *prefix)
 		break;
 	}
 
+	if (whitespace) {
+		struct strbuf buf2 = STRBUF_INIT;
+
+		strbuf_reset(&buf);
+		strbuf_reset(&buf2);
+		strbuf_addf(&buf2, "--%s", whitespace);
+		strbuf_addstr(&buf, "--whitespace");
+		sq_quote_buf(&buf, buf2.buf);
```

Comment:
Please do keep in mind that `add_var()` *already* sq-quotes the values. So there is no need to do that here, either.